projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -70,8 +88,20 @@ export class TabParagraphContainerComponent
     )
   );
 
-  select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+  select(tabNum: number, event?: Event): void {
```

Comment:
It would be good to use the aproppriate event type, i.e. `MouseEvent` for the click event.